profiles/models.py

```diff
@@ -239,16 +239,30 @@ class VerifiedPersonalInformation(ValidateOnSaveModel, NullsToEmptyStringsModel)
     profile = models.OneToOneField(
         Profile, on_delete=models.CASCADE, related_name="verified_personal_information"
     )
-    first_name = fields.EncryptedCharField(max_length=1024, blank=True)
-    last_name = fields.EncryptedCharField(max_length=1024, blank=True)
-    given_name = fields.EncryptedCharField(max_length=1024, blank=True)
+    first_name = fields.EncryptedCharField(
+        max_length=1024, blank=True, help_text="First name(s)."
+    )
+    last_name = fields.EncryptedCharField(
+        max_length=1024, blank=True, help_text="Last name."
+    )
+    given_name = fields.EncryptedCharField(
+        max_length=1024, blank=True, help_text="The name the person is called with."
+    )
     national_identification_number = fields.EncryptedCharField(
-        max_length=1024, blank=True
+        max_length=1024,
+        blank=True,
+        help_text="Can be social security number or other person identifier.",
```

Comment:
I think this should just read National identification number. As "social security number" hasn't been used in decades.

Reply:
Same thing as with the country code. Originally these fields were very "under specified". But they will be much stricter in near future. Perhaps revisit documentation then?

Reply:
I would still add the text as truthful as possible right away.

Reply:
Can do.

Reply:
Changed in 0d6bf75947.